src/explore-education-statistics-frontend/src/services/dataSetFileService.ts

```diff
@@ -86,6 +87,7 @@ export interface DataSetFileSummary {
   lastUpdated: string;
   api?: DataSetFileApi;
   meta: {
+    numDataFileRows: number;
```

Comment:
Should this also be optional, or are we just letting it set to zero  if nothing's returned from the backend for this field?